programs/ziti-prox-c/proxy.c

```diff
@@ -590,6 +591,12 @@ void run(int argc, char **argv) {
         model_map_set(&app_ctx.listeners, service_name, l);
     }
 
+    ziti_config cfg;
+    ziti_context ztx;
+
+    ziti_load_config(&cfg, config);
+    ziti_context_init(&ztx, &cfg);
+
     ziti_options opts = {
             .config = config,
```

Comment:
should the config string be present in the opts if the config is being passed as an object through the context?

Reply:
it will be ignored anyway, and field will be removed in the future

Reply:
Since this app serves as a clear example, would it make sense to remove it here?